app-localize-behavior.html

```diff
@@ -227,6 +232,12 @@
     };
   },
 
+  __languageChanged: function(newLang, oldLang) {
+    if (newLang === oldLang) return;
+    this.language = newLang;
+    this.fire('app-localize-behavior-language-changed', newLang);
```

Comment:
So far I can't get it to work with notify: true. I suspect this is due to:
"When the property changes, the element fires a non-bubbling DOM event to indicate those changes to interested hosts."
Does non-bubbling here mean that only the parent element will receive it?
I need it to go farther up the hierarchy.
Unless there is a way to configure a notify to send a bubbling event, maybe I need to fire the event myself.